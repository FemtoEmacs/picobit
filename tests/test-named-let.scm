(let loop ((x 0))
  (if (< x 10)
      (begin (display x)
	     (loop (+ x 1)))))
