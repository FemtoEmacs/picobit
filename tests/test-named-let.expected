:0080000080
:10800000FBD71D0281194000810F4000810C400008
:10801000810A4000810B4000804200028027011647
:1080200080270117802401188071011A806E011BBE
:108030008027011C8077011D8073011E80240002AF
:10804000806A00028073011F806701208040011058
:1080500080400111803201128027011380660114D3
:108060008069011580780002807D010D8066010E17
:10807000807001098078010902B401975002B401AF
:1080800091510002B4008A61EED80120B0018C0247
:10809000B400947101202102CDB400AEB58AEE0483
:1080A00021B581710120CEEF022022D000EF012006
:1080B0000EC8B3018A20B588EE2005C122B56571CE
:1080C0000120DAB8954121DBB6800221CC9A21CE7D
:1080D0002161EE2022CFB67200EF20B6800120DAB7
:1080E0009EA02607E620B558EEA02607E600EF2062
:1080F000C0B8BF20B582B648012004C8B8AEA031D0
:1081000021B5A402B583CDDCEF02A034210EC5C198
:1081100022CD2102CDB401197101210EC89220EFA8
:10812000210E406221B662010421C2EF2002B5593E
:10813000B75520CCB8B8A02C07E620CEB51FEE204E
:10814000CFB6800120D296A02D07E600EF20CC9D6F
:10815000A02407E620CEB505EE20CFB666A106B076
:1081600000C0EE20B000DDEEA02D07E600EF20D924
:1081700095A107B100C020F09C2095A105B100C0D9
:10818000A104B100C0A108B100C000EF012002CDE0
:10819000EF012007E600EF022004C8212302CDCD25
:1081A000B401A471012204C822242302CDCDCDB490
:1081B00001B3710123B8DC21B0012723222602CDAF
:1081C000CDCDB401C6710122B8C621B0012721244A
:1081D0002602CDCDCDB401D971012022242602CDB5
:1081E000CDCDCDB401E770002121C4232502CDCD32
:1081F000B401F4710122B893219220EF22980194E6
:1082000020B1012720EF21B87BB77500B86EB76A9F
:0682100021B73B21B72558
:00000001FF
